File: cpp/455_分发饼干_easy.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        sort(g.begin(), g.end());
        sort(s.begin(), s.end());
        int i=0, j=0;
        int num_g = g.size();
        int num_s = s.size();
        while(i<num_g && j<num_s)
            if(s[j]>=g[i]){
                i++;
                j++;
            }
            else
                j++;
        return i;
    }
};

File: python/455_分发饼干_easy.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        g, s = sorted(g), sorted(s)
        i, j = 0, 0
        while i<len(g) and j<len(s):
            if s[j] >= g[i]:
                i, j = i+1, j+1
            else:
                j+=1
        return i